client/extensions/wp-super-cache/preload-tab.jsx

```diff
@@ -166,11 +166,11 @@ class PreloadTab extends Component {
 									) }
 
 									{ translate(
-										'(minimum %d minute)',
-										'(minimum %d minutes)',
+										'(minimum %(minutes)d minute).',
+										'(minimum %(minutes)d minutes).',
 										{
-											args: minimum_preload_interval,
-											count: minimum_preload_interval,
+											args: { minutes: minimum_preload_interval || 0 },
+											count: minimum_preload_interval || 0,
```

Comment:
In this case the field is actually missing from the API altogether (known issue still to be fixed), so it's `undefined` and setting to 0 in the destructuring assignment would work, but to be safe I'll leave it as is in case it ever decides to return `null`. :|